File: Lib/Telerik/Examples/QuickStart/Editors/EditorExampleForm.cs
using System.Drawing;
using System.Windows.Forms;
using Telerik.QuickStart.WinControls;
using Telerik.WinControls;

namespace Telerik.Examples.WinControls.Editors
{
    /// <summary>
    /// This is a base control for all RadComboBox examples.
    /// </summary>
    public partial class EditorExampleBaseForm : ExamplesForm
    {
        private bool centerPanel = false;

        public EditorExampleBaseForm()
        {
            InitializeComponent();

            this.radPanelDemoHolder.PanelElement.PanelFill.Visibility = ElementVisibility.Collapsed;
            this.radPanelDemoHolder.PanelElement.PanelBorder.Visibility = ElementVisibility.Collapsed;
            this.radPanelDemoHolder.RootElement.EnableElementShadow = false;
            this.radPanelDemoHolder.Dock = DockStyle.Fill;
            this.radPanelDemoHolder.Padding = new Padding(0);
            this.Padding = new Padding(0);
        }

        public bool CenterPanel
        {
            get
            {
                return this.centerPanel;
            }
            set
            {
                this.centerPanel = value;
                if (this.centerPanel)
                {
                    this.CenteredControl = this.radPanelDemoHolder;
                }
                else
                {
                    this.CenteredControl = null;
                    this.radPanelDemoHolder.Location = Point.Empty;
                }
            }
        }
    }
}

File: Lib/Telerik/Examples/QuickStart/Editors/EditorExampleForm.vb
Imports System
Imports System.Drawing
Imports System.Windows.Forms
Imports Telerik.QuickStart.WinControls
Imports Telerik.WinControls

Namespace Telerik.Examples.WinControls.Editors
    Partial Public Class EditorExampleBaseForm
        Inherits ExamplesForm

        Private centerPanel_Field As Boolean = False

        Public Sub New()
            InitializeComponent()
            Me.radPanelDemoHolder.PanelElement.PanelFill.Visibility = ElementVisibility.Collapsed
            Me.radPanelDemoHolder.PanelElement.PanelBorder.Visibility = ElementVisibility.Collapsed
            Me.radPanelDemoHolder.RootElement.EnableElementShadow = False
            Me.radPanelDemoHolder.Dock = DockStyle.Fill
            Me.radPanelDemoHolder.Padding = New Padding(0)
            Me.Padding = New Padding(0)
        End Sub

        Public Property CenterPanel As Boolean
            Get
                Return Me.centerPanel_Field
            End Get
            Set(ByVal value As Boolean)
                Me.centerPanel_Field = value

                If Me.centerPanel_Field Then
                    Me.CenteredControl = Me.radPanelDemoHolder
                    Me.CenterControl()
                Else
                    Me.CenteredControl = Nothing
                    Me.radPanelDemoHolder.Location = Point.Empty
                End If
            End Set
        End Property
    End Class
End Namespace
